### ISSUE TITLE
Dynamically change SecureMaskedInput mask, similary to MaskedInput

### ISSUE THREAD

**Issue by nkosig-jemstep** [rapsalands/react-control-library#6]
Ability to change mask on SecureMaskedInput control. The MaskedInput control is able to do this with no issues.

Use case:
Phone number mask should change dynamically depending on weather local/international number is expected.

```
  const [data, setData] = React.useState("");
  const n = /^[0-9]*$/;
  const getPhoneMask = (isInternational: boolean) => isInternational ? ["+", n, n, n, n, n, n, n, n, n, n] : ["(", n, n, n, ")", " ", n, n, n, "-", n, n, n, n];

  return (
      <SecureMaskedInput
        mask={getPhoneMask(isInternational)}
        secure={{
          getValue: (detail, data) => {
            if (detail.value.length < 10) {
              return data;
            }
            return isInternational ?
            `+******${detail.value.substring(6, 10)` :
            `(***) ***-${detail.value.substring(6, 10);
          }
        }}
        exactLength={10}
        value={data}
        onChange={(e) => setData(e.target.value)}
      />
  );
```

**Comment by nkosig-jemstep** [comment]
Thank you for the prompt response and fix. The provided CodeSandbox works for my use case and version 1.2.9 resolves the issue.

I'll make sure to provide CodeSandbox link next time :)

**Comment by rapsalands** [comment]
Issue resolved on version 1.2.9 update.
